### PR TITLE
Fix set invalid instance UID error throw

### PR DESCRIPTION
**Pull request by sairam866**
Fixing error throw logger when setting invalid instance UID provided by the server.

### CODE REVIEW

**Comment by tigrannajaryan** [comment]
Thank you for the fix @sairam866 
Please sign the CLA so that we can accept the PR.

**Comment by sairam866** [comment]
@tigrannajaryan Updated my commit with my company email. CLA should be fine now.

File: client/internal/receivedprocessor.go [hunk 1]
```diff
@@ -217,7 +217,7 @@ func (r *receivedProcessor) rcvAgentIdentification(agentId *protobufs.AgentIdent
 
 	err := r.sender.SetInstanceUid(agentId.NewInstanceUid)
 	if err != nil {
-		r.logger.Errorf("Error while setting instance uid: %v, err")
+		r.logger.Errorf("error while setting instance uid: %v", err)
```
Comment (tigrannajaryan) [inline client/internal/receivedprocessor.go:220]: We capitalize log messages:
```suggestion
		r.logger.Errorf("Error while setting instance uid: %v", err)
```

Comment (sairam866) [reply client/internal/receivedprocessor.go]: Noted. Updated the commit.

**Review by tigrannajaryan — APPROVED** [review]

**Comment by tigrannajaryan** [comment]
Thanks @sairam866

**Pull request merged**